be/src/olap/compaction.cpp

```diff
@@ -1122,6 +1122,8 @@ Status CloudCompactionMixin::construct_output_rowset_writer(RowsetWriterContext&
     ctx.compaction_level =
             _engine.cumu_compaction_policy(compaction_policy)->new_compaction_level(_input_rowsets);
 
+    ctx.write_file_cache = true;
```

Comment:
make it a config option.
base compaction may flush the cache, which cause performance issue